partner_name_vat/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Eric Antones <eantones@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Partner name with vat",
+    "summary": "This module concatenates vat number to partner name on display_name",
+    "version": "12.0.02.03.21",
```

Comment:
This should be 12.0.1.0.0 on the first commit of the migrations as it's described here https://github.com/nuobit/odoo-addons/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration.
@VrajaTechnologies can you please change it and push a new commit